### ISSUE TITLE
Segmentation fault when processing paramPoly3 geometries

### ISSUE THREAD

**Issue by michikommader** [DLR-TS/gdal#2]
"the GDAL error message Segmentation fault (core dumped) appears only when the input xodr file has paramPoly3 geometries.", see DLR-TS/xodr#1

**Comment by michikommader** [comment]
DLR-TS/xodr#1

**Comment by cristhianmurcia182** [comment]
Hi, the error appears when the paramPoly3 object does not contain the pRange attribute. The pRange attribute is optional, and the current version of the code located in this repository assumes that all the paramPoly3 instances have this attribute. The code has been corrected by checking whether the pRange value is available if that is not the case, we define this parameter by hand in the range [0;1], as stated in the OpenDrive 1.4. documentation. After making this changes, I executed the code, with the /KA-Suedtangente-atlatec-Roadshape.xodr, obtaining appropriate results. I will update the code soon so that this error does not occur again.
Kind Regards, sorry for the inconveniences.
